Content.Server/Temperature/Systems/TemperatureSystem.cs

```diff
@@ -160,7 +160,8 @@ private void OnAtmosExposedUpdate(EntityUid uid, TemperatureComponent temperatur
         var heatCapacity = GetHeatCapacity(uid, temperature);
         var heat = temperatureDelta * (airHeatCapacity * heatCapacity /
                                        (airHeatCapacity + heatCapacity));
-        ChangeHeat(uid, heat * temperature.AtmosTemperatureTransferEfficiency, temperature: temperature);
+        ChangeHeat(uid, heat * temperature.AtmosTemperatureTransferEfficiency, temperature: temperature); // move heat to entity
+        _atmosphere.AddHeat(args.GasMixture, -1 * heat * temperature.AtmosTemperatureTransferEfficiency); // remove heat from air
```

Comment:
Separate PR, because this is a bug fix to `TemperatureSystem`.

Reply:
Its necessary for fires.
Without it, temperatures cant equalize and run away.